### PR TITLE
Navigation_Dev

### PR DESCRIPTION
**Pull request by collinnolen**

### CODE REVIEW

**Comment by collinnolen** [comment]
Added Land, and updated player movement.

File: Assets/Scripts/Navigation/KeyBoardManager.cs [hunk 1]
```diff
@@ -4,39 +4,41 @@
 
 public class KeyBoardManager : MonoBehaviour {
```
Comment (artsiriwatt) [inline Assets/Scripts/Navigation/KeyBoardManager.cs]: Formatting of this file is messed up, make sure you fix that

File: Assets/Scripts/Navigation/Map.cs [hunk 2]
```diff
@@ -4,148 +4,119 @@
 
 public class Map : MonoBehaviour {
```
Comment (artsiriwatt) [inline Assets/Scripts/Navigation/Map.cs]: Formatting of this class is messed up

**Comment by artsiriwatt** [comment]
https://forum.unity.com/threads/monodevelop-changing-tab-size-has-no-effect.155365/#post-1099202

Tools > Options > Text Editor > Markers and Rulers > Show Invisible Characters > Selection (Use all three)

File: Assets/Scripts/Navigation/KeyBoardManager.cs (hunk 1 above)
Comment (collinnolen) [reply Assets/Scripts/Navigation/KeyBoardManager.cs]: Should be fixed

File: Assets/Scripts/Navigation/Enemy.cs [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        }
+
+        FinishMove();
+    }
+
+    private void FinishMove()
+    {
+        //Temp conditional
+        //TODO create conditional to check if defeated to skip adding to the turnQueue
+        if (true)
```
Comment (artsiriwatt) [inline Assets/Scripts/Navigation/Enemy.cs]: remove the brackets for style consistency

File: Assets/Scripts/Navigation/Hex.cs [hunk 4]
```diff
@@ -10,6 +10,10 @@ public class Hex : MonoBehaviour {
     public int x;
     public int y;
 
+    public List<GameObject> occupents;
```
Comment (artsiriwatt) [inline Assets/Scripts/Navigation/Hex.cs]: should be `occupants` (unless the misspelling was intentional)

**Comment by artsiriwatt** [comment]
Continued in PR #22, which was merged; closing

**Pull request closed without merging**